compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/CallableReferenceLowering.kt

```diff
@@ -503,7 +520,7 @@ internal class CallableReferenceLowering(val context: JvmBackendContext) : FileL
                 val irBuilder = context.createIrBuilder(function.symbol, startOffset, endOffset)
                 body = irBuilder.irBlockBody(startOffset, endOffset) {
                     +irReturn(
-                        IrConstImpl.string(-1, -1, context.irBuiltIns.stringType, callee.name.asString())
+                        IrConstImpl.string(-1, -1, context.irBuiltIns.stringType, callee.originalName.asString())
```

Comment:
Probably, the same change will be useful in `PropertyReferenceLowering`. However, I don't fully understand how is it relevant for inline classes?

Reply:
Right now it's not needed for `PropertyReferenceLowering`, because it runs before inline class lowering and already uses descriptors to get the property name. If you want I can change it, though.